README.md

```diff
@@ -28,8 +28,8 @@ Those rewards can be transferred to the `skimRecipient`.
 The vault's owner has the choice to distribute back these rewards to vault depositors however they want.
 For more information about this use case, see the [Rewards](#rewards) section.
 
-All actions that may be against users' interests (e.g. enabling a market with a high exposure, increasing the fee) are subject to a timelock of minimum 12 hours.
-If set, the `guardian` can revoke the action during the timelock except for the fee increase.
+All actions that may be against users' interests (e.g. enabling a market with a high exposure) are subject to a timelock of minimum 24 hours.
+If set, the `guardian` can revoke the action during the timelock.
```

Comment:
The `guardian` (or the `owner` itself because he's also a guardian) can always remove the action. `revokePending*` will never revert.